packages/app/src/components/InputField.vue

```diff
@@ -0,0 +1,144 @@
+        <!-- Header	-->
+        <div class="flex-100">
+          <h1 class="font-bold text-3xl flex-100 mb-8">Gimme a new Pet</h1>
+        </div>
+        <!--form-->
+        <div class="flex flex-100 mt-8">
+          <form class="flex-100" @submit.prevent="createPet">
+            <!--inputs-->
+            <div class="flex flex-100">
+              <div class="flex-grow mr-2">
+                <label class="block mb-1 font-bold uppercase" for=""
+                  >Name</label
+                >
+                <input
+                  v-model="form.name"
+                  class="bg-gray-200 w-full rounded-lg py-3 px-3 shadow-inner"
+                  type="text"
+                />
+              </div>
+              <div class="flex-grow ml-2">
+                <label class="block mb-1 font-bold uppercase" for=""
+                  >Breed</label
+                >
+                <input
+                  v-model="form.breed"
+                  class="bg-gray-200 w-full rounded-lg py-3 px-3 shadow-inner"
+                  type="text"
+                />
+              </div>
+            </div>
+            <div class="flex flex-100 mt-4">
+              <div class="flex-grow mr-2">
+                <label class="block mb-1 font-bold uppercase" for=""
+                  >Date of Birth</label
+                >
+                <input
+                  v-model="form.dateOfBirth"
+                  class="bg-gray-200 w-full rounded-lg py-3 px-3 shadow-inner"
+                  type="date"
+                />
+              </div>
+            </div>
+
+            <div class="flex flex-100 mt-4">
+              <input
+                type="submit"
+                value="Save"
+                class="bg-pink-dribbble flex-50 text-white relative py-4 rounded"
+              />
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, onMounted, ref, reactive } from 'vue';
+import { Pet } from '@project-leek/commons';
+import feathers from '../lib/feathers';
+
+export default defineComponent({
+  name: 'InputField',
+  setup() {
+    const form = reactive({
+      name: '',
+      breed: '',
+      dateOfBirth: new Date(),
+    });
+
+    const pets = ref<Pet[]>([]);
+
+    onMounted(async () => {
+      const response = await feathers.service('pets').find();
+      pets.value = response.data;
+    });
+
+    const createPet = async () => {
+      // Warum geht der Konstruktor nicht?
+      // const petti = new Pet(form.name, form.breed, form.dateOfBirth);
+      const petti: Pet = {
+        name: form.name,
+        breed: form.breed,
+        dateOfBirth: form.dateOfBirth,
+      };
+
+      await feathers
+        .service('pets')
+        .create(petti)
+        .then(() => {
+          form.name = '';
+          form.breed = '';
+          form.dateOfBirth = '';
+        });
```

Comment:
This is somehow duplicated as you use `.then` and `await`. We should generally only use `await` as it is more readable.

```suggestion
      await feathers.service('pets').create(petti);
      form.name = '';
      form.breed = '';
      form.dateOfBirth = '';
```

Reply:
I had an error with the unuiqe Identifier and did not know if I deleted the values befor writing the Data with feathers. Now I know better, and can rewrite this code.

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317